pkg/beyla/config.go

```diff
@@ -45,6 +45,7 @@ const (
 var DefaultConfig = Config{
 	ChannelBufferLen: 10,
 	LogLevel:         "INFO",
+	ShutdownTimeout:  5 * time.Minute,
```

Comment:
I'm not sure what is the best default time to set, left it pretty big imho

Reply:
It should be few seconds (maybe 10?). Actually Kubernetes would kill Beyla before that timeout.

Reply:
okay, changed it to 10 seconds, thanks